examples/Huggingface_Transformers/Pipeline_handler_generalized.py

```diff
@@ -0,0 +1,178 @@
+'''
+TODO:
+(i) Add support for remaining Pipeline tasks
+(ii) Add support for Captum explanations
+
+'''
+
+import os
+import torch
+import base64
+import io
+from PIL import Image
+
+from ts.torch_handler.base_handler import BaseHandler
+from transformers import pipeline
+
+
+#Reference: https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.pipeline.task
+pipeline_supported_tasks= [
+    "image-classification",
+    "sentiment-analysis",
+    "text-classification"
+]
+
+task="<PLACEHOLDER>"
```

Comment:
As mentioned [above](https://github.com/pytorch/serve/pull/1822#discussion_r963959315), it might be done in another way and feel free to do so later.